bin/light-base/src/runtime_service.rs

```diff
@@ -836,18 +837,21 @@ pub struct SubscribeAll {
 pub enum Notification {
     /// A non-finalized block has been finalized.
     Finalized {
-        /// Blake2 hash of the block that has been finalized.
+        /// Blake2 hash of the header of the block that has been finalized.
         ///
         /// A block with this hash is guaranteed to have earlier been reported in a
         /// [`BlockNotification`], either in [`SubscribeAll::non_finalized_blocks_ancestry_order`]
         /// or in a [`Notification::Block`].
         ///
-        /// It is, however, not guaranteed that this block is a child of the previously-finalized
-        /// block. In other words, if multiple blocks are finalized at the same time, only one
+        /// It is also guaranteed that this block is a child of the previously-finalized block. In
+        /// other words, if multiple blocks are finalized at the same time, only one
```

Comment:
This was actually already the case. The difference with this PR is that it is now a guarantee and not an implementation detail.